evalai/utils/auth.py

```diff
@@ -76,6 +76,37 @@ def get_user_auth_token():
         sys.exit(1)
 
 
+def validate_user_auth_token_by_profile(auth_token):
+    """
+    Checks tokens availability via server.
+    """
+    url = "{}{}".format(get_host_url(), URLS.profile.value)
+    try:
+        headers = {"Authorization": "Token {}".format(auth_token)}
+        response = requests.post(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        if response.status_code in EVALAI_ERROR_CODES:
+            echo(
+                style(
+                    "Failed: Unable to validate token.",
```

Comment:
Hi @takitsuse 
Thanks for prompt changes on reviews!
What do you think of using
```sh
Error: Invalid Auth Token (unrecognized or wrong Auth Token)
```
instead of 
```sh
Failed: Unable to validate token.
```
as I feel that `Failed: Unable to validate token` may also express to the user that there is an issue connecting to server and not invalid token error.

Reply:
Hi @vkartik97! The error message that you suggested is obviously better than mine, so I will do use it.